man/systemd.unit.xml

```diff
@@ -1031,6 +1033,15 @@
         whether a certain path exists, is a regular file and marked
         executable.</para>
 
+        <para><varname>ConditionUID=</varname> takes a numeric
+        <literal>UID</literal>. This condition may be used to check
+        wether the service manager is running as the given user id.
+        This option is only meaningful for the user managers.</para>
```

Comment:
it might make sense to clarify that this is about the "real" UID, rather than the "effective" or "saved" or even "fs" UID...